transactions/WFLY-14869-microprofile-lra.adoc

```diff
@@ -0,0 +1,106 @@
+= Support for MicroProfile LRA
+:author:            Ondrej Chaloupka
+:email:             ochaloup@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          transaction, saga, lra, microprofile
+
+== Overview
+
+The goal of this RFE is providing the Galleon feature pack for WildFly
+which integrates the https://github.com/jbosstm/narayana/tree/5.11.2.Final/rts/lra[Narayana implementation]
```

Comment:
Can change to subsequent versions if we need to update the narayana code?

Reply:
My point was to show where the LRA implementation can be found. I wanted to have a static link that will be working for anybody coming to the document in the next years despite of the changes in the Narayana repository.
The link can be changed to anything where the LRA code will be placed.

Reply:
okay, except that [linking to master](https://github.com/jbosstm/narayana/tree/master/rts/lra) would server the same purpose.

Reply:
I see, I have a different point of thinking but as it's better to use the `master` for you I'm changing it.